pages/linux/faketime.md

```diff
@@ -0,0 +1,11 @@
+# faketime
+
+> Fake this computers time for a given command.
```

Comment:
Please also add a line for `Homepage: <>`. And how about `Fake the system time for a given command` ?

Reply:
What exactly would you suggest for the homepage? There is a [GitHub repo for the library](https://github.com/wolfcw/libfaketime), but nothing for the `faketime` tool itself, except for [online manpages](http://manpages.ubuntu.com/manpages/trusty/man1/faketime.1.html).

Reply:
I think the manpage is fine.

Reply:
Duh .. thanks @mebeim